### ISSUE TITLE
Dates are old on Change Log entries 9, 10, 11

### ISSUE THREAD

**Issue by mpawlow** [IBMDataScience/datascix#14]
### Problem

The latest entries do not show up on the top
See: https://apsportal.ibm.com/whats-new?context=analytics

**Comment by mpawlow** [comment]
FYI @msochka @aruizga7

**Comment by msochka** [comment]
@mpawlow The new entries had dates from 2016-08-24 (other entries had later dates). Those are the dates of the blog posts and I think they are showing up in the correct order. But let me know for sure. I am only just getting used to this setup. From @aruizga7 [Slack DM, 2016-11-17] "can you modify the dates so they match with the blog post dates pls?"

Also, the entry in question did not have a blog entry for it. Only a picture. Is there a blog about making a favorite? I did not find one.

**Comment by mpawlow** [comment]
@msochka Got it. 
Sorry, I thought these new Change Log entries corresponded with new blog posts.

> Is there a blog about making a favorite?

Probably not.
I figured it might have been an oversight.
If it is an acceptable use case to have new entries without corresponding blog post URLs, the DEV team will need to update the app to hide the button in these circumstances.
@aruizga7 will need to confirm this requirement.

Closing this Issue as `invalid`
